src/SassBinary.php

```diff
@@ -133,10 +133,10 @@ public function getBinaryName(): string
 
         if (str_contains($os, 'linux')) {
             if ('arm64' === $machine || 'aarch64' === $machine) {
-                return $this->buildBinaryFileName('linux-arm64');
+                return $this->buildBinaryFileName('linux-musl-arm64');
```

Comment:
That seems wrong to me... all linux should probably not use the musl version, or did i miss something ?

Reply:
yeah, sounds fair.. I suppose we need to use `musl` only for Alpine? Any ideas on how to check we're on exactly Alpine Linux distribution?

Reply:
That's what i'm looking for ... with no results for now.

Nothing in `php_uname` nothing in `$_ENV` or `$_SERVER ` ... maybe a configuration key ? :|

Reply:
ChatGPT suggests something like `file_exists('/etc/alpine-release')` - not sure 100% but it may work I suppose?

Reply:
I created a php:8.3-alpine docker image and downloaded this version: `dart-sass-1.69.6-linux-musl-arm64`

The binary works perfectly.

```php
string(5) "linux"
string(7) "aarch64"
string(5) "Linux"
string(12) "462bf4f33866"
```

So remains only the "detection" part (because i'm not sure if the arm64 will work on anything but a macos host)

Reply:
The `/etc/alpine-release works` !

Reply:
The following seems to work ... i'd rather have someone using Alpine test a bit more :) 

```php
        if (str_contains($os, 'linux')) {
            $baseName = file_exists('/etc/alpine-release') ? 'linux-musl' : 'linux';
            if ('arm64' === $machine || 'aarch64' === $machine) {
                return $this->buildBinaryFileName($baseName.'-arm64');
            }
            if ('x86_64' === $machine) {
                return $this->buildBinaryFileName($baseName.'-x64');
            }

            throw new \Exception(sprintf('No matching machine found for Linux platform (Machine: %s).', $machine));
        }
```